Header header

uint32 src_img_msg_seq

bool found_right_border
bool found_left_border

int8 num_lanes
int8 current_lane

float64[] lanes_start_offset
float64[] line_right_border
float64[] line_left_border

float64[] line_latest_valid_right_border
float64[] line_latest_valid_left_border
